lib/src/layer/polyline_layer.dart

```diff
@@ -138,10 +282,11 @@ class PolylinePainter extends CustomPainter {
       paint = Paint();
     }
 
-    final origin = map.project(map.center).toOffset() - map.size.toOffset() / 2;
+    final origin =
+        camera.project(camera.center).toOffset() - camera.size.toOffset() / 2;
```

Comment:
origin could be cached and later used in `hitTest`? but up to you.

Reply:
It's possible I suppose but more effort than reward - having to keep track of it and change it in `shouldRepaint` and also more space for bugs to sneak in.

Reply:
It's **not** possible. `paint` and `hitTest` have completely independent lifecycles. You can hit test without repainting and you can repaint w/o hit testing.